rascal-vscode-extension/src/auto-jvm/downloaders.ts

```diff
@@ -211,7 +215,7 @@ export async function fetchAndUnpackCorretto(arch: CorrettoArchitectures, platfo
 
 export async function fetchAndUnpackTemurin(arch: TemurinArchitectures, platform: TemurinPlatforms, jdk11Release: string, mainJVMPath: string, progress: ProgressFunc): Promise<string> {
     // https://api.adoptium.net/v3/binary/version/jdk-11.0.13%2B8/linux/x64/jdk/hotspot/normal/eclipse?project=jdk
-    const url = `https://api.adoptium.net/v3/binary/version/jdk-${encodeURIComponent(jdk11Release)}/${platform}/${arch}/jdk/hotspot/normal/eclipse?project=jdk`;
+    const url = `https://api.adoptium.net/v3/binary/version/${encodeURIComponent(jdk11Release)}/${platform}/${arch}/jdk/hotspot/normal/eclipse?project=jdk`;
```

Comment:
perhaps rename the variable? jdk11Release to jdkRelease?

Reply:
I though I got them all. I'll take care of this one as well.